api/openapi.yaml

```diff
@@ -167,6 +176,28 @@ paths:
           $ref: "#/components/responses/BadRequestProblemResponse"
         default:
           $ref: "#/components/responses/UnexpectedProblemResponse"
+  /api/v1alpha2/namespaces:
+    post:
+      description: Create namespace
+      operationId: createNamespace
+      tags:
+        - Meters
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Namespace"
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Namespace"
+          $ref: "#/components/responses/NotFoundProblemResponse"
```

Comment:
Must be left here by accident. Please also include in case no manager is used:
```
        "501":
          $ref: "#/components/responses/NotImplementedProblemResponse"
```